app/models/stylesheet_cache.rb

```diff
@@ -42,6 +42,10 @@ def self.add(target, digest, content, source_map, max_to_keep: nil)
     end
   end
 
+  def self.clean_up
+    StylesheetCache.where('created_at < TIMESTAMP ?', 150.days.ago).delete_all
```

Comment:
```suggestion
    StylesheetCache.where('created_at < ?', 150.days.ago).delete_all
```

I'm not sure what does `TIMESTAMP` do but we don't need it because ActiveRecord will handle the conversion.

Reply:
I also think we should extract `150` days to a constant so that it doesn't have to be duplicated in tests.